README.md

```diff
@@ -70,108 +88,118 @@ reusing the configuration from the local disk. The process is:
 The options for the `docker run...` command are:
 
 - `-d` - Detached mode: Unifi-in-Docker runs in the background
-- `--init` - Recommended to ensure processes get reaped
+- `--init` - Recommended to ensure processes get reaped when they die
 - `--restart=unless-stopped` - If the container should stop for some reason,
 restart it unless you issue a `docker stop ...`
-- `-p 8443:8443` - Set the ports to pass through to the container.
-The list above is the minimal set for a working Unifi Controller. 
-- `-e TZ=...` Set an environment variable named `TZ` with the desired time zone
-([list of timezones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones))
+- `-p ...` - Set the ports to pass through to the container.
+The list above - `-p 8080:8080 -p 8443:8443 -p 3478:3478/udp`
+is the minimal set for a working Unifi Controller. 
+- `-e TZ=...` Set an environment variable named `TZ` with the desired time zone.
+Find your time zone in this 
+[list of timezones.](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)
 - `-e ...` See the [Environment Variables](#environment-variables)
 section for more environment variables.
-- `-v ...` - Bind the volume `~/unifi` on the host to the directory `/unifi`inside the container.
+- `-v ...` - Bind the volume `~/unifi` on the Docker host
+to the directory `/unifi`inside the container.
+**These instructions assume you placed the "unifi" directory in your home directory.**
+If you created the directory elsewhere, modify the `~/unifi` part of this option to match.
 See the [Volumes](#volumes) discussion for other volumes used by Unifi Controller.
 - `--user unifi` - Run as a non-root user. See the [Run as non-root User](#run-as-non-root-user) discussion below
-- `jacobalberty/unifi:7` - the name of the container to use.
-The `jacobalberty...` image comes from [Dockerhub.](https://hub.docker.com/r/jacobalberty/unifi)
+- `jacobalberty/unifi` - the name of the container to use.
+The `jacobalberty...` image is retrieved from [Dockerhub.](https://hub.docker.com/r/jacobalberty/unifi)
 See the discussion about [Supported Tags](#supported-docker-hub-tags-and-respective-dockerfile-links) below.
 - **Windows Users:** There is a problem with Mongo database and a bind-mount volume.
-You can keep data locally but should use `-v unifi:/unifi` (no leading `~/`)
-**_IS THIS TRUE?_** See [Mongo and Docker for Windows](#mongo-and-docker-for-windows) for more details.
-
-## Supported Docker Hub Tags and Respective `Dockerfile` Links
+See [Mongo and Docker for Windows](#mongo-and-docker-for-windows) for more details.
 
-### "latest" tag
+## Supported Tags
 
-`latest` now tracks unifi 7.1.x as of 2022-05-18.
+_[**Note:** Docker's rule is to retrieve `:latest` if no tag is specified.
+Newcomers should not be using an RC version,
+so I suggest that we tag the most recent stable version with `:latest`.
+Someone who wants the release candidate can add the `:development` tag -richb]_
```

Comment:
The way I handle this is `:latest` only contains the latest stable,  i have rcs under the `:rc` tag so its safe to use `:latest` for if you trust ubiquiti's definition of stable, though with unifi it is probably actually better to use a specific version tag like `:v7.1` for anything beyond home personal use

Reply:
I agree. Perhaps we should only have `:latest` and `:7.1.66` tags (that is, the full version number, not `v7` nor `v7.1`)

Reply:
... and to follow along with your thought, we should tell people to use the `:rc` tag if they want an RC. (Not my newly-invented `:development` tag...